Ports/neofetch/patches/0001-Add-support-for-serenity.patch

```diff
@@ -71,8 +71,10 @@ index 1e4b564..d2a12cd 100755
      case $os in
 -        Linux|Windows|MINIX)
 +        Linux|Windows|MINIX|SerenityOS)
-             if [[ -r /proc/uptime ]]; then
-                 s=$(< /proc/uptime)
+-            if [[ -r /proc/uptime ]]; then
+-                s=$(< /proc/uptime)
++            if [[ -r /sys/kernel/uptime ]]; then
```

Comment:
I think this patch in neofetch needs re-done. We overwrite Linux and Windows and Minix behavior with our own.